lib/workers/global/config/parse/index.ts

```diff
@@ -39,7 +39,7 @@ export async function parseConfigs(
   }
 
   if (!config.privateKey && config.privateKeyPath) {
-    config.privateKey = await readFile(config.privateKeyPath);
+    config.privateKey = (await readFile(config.privateKeyPath)).toString();
```

Comment:
```suggestion
    config.privateKey = await readFile(config.privateKeyPath, 'utf8');
```

Read as string, no buffer required 😉  needs snapshot update